### DISCUSSION TITLE
internet not working

### DISCUSSION THREAD

**Discussion by harrylolzxe** [mitmproxy/mitmproxy#6835] [Q&A]
ive seen another post about this, but it didn't help me. if i have my mimt proxy closed, i can not access the internet. could someone help me?

**Comment by harrylolzxe** [comment]
im on win 11 ^, used mitm 3-4 days ago

**Comment by emanuele-em** [comment]
I think the only thing could be that you left traffic redirection set to proxy, check in network settings if you have redirection to 127.0.0.1:8080 of both http and https traffic

[how to proxy on windows](https://support.microsoft.com/en-us/windows/use-a-proxy-server-in-windows-03096c53-0554-4ffe-b6ab-8b1deee8dae1)

**Reply by harrylolzxe** [reply]
you mean, in the windows settings, right?

**Reply by emanuele-em** [reply]
yes

**Reply by harrylolzxe** [reply]
i couldn't find it there

**Reply by harrylolzxe** [reply]
i dont have any redirection ^

**Reply by emanuele-em** [reply]
Ok then I don't think mitmproxy is responsible

**Comment by harrylolzxe** [comment]
sorry if im wrong ^ im new to this and i only installed mitm for another app

**Comment by harrylolzxe** [comment]
now that i think about it , i dont think you mean windows settings, but just let me know what you meant
